test/file.jl

```diff
@@ -1692,3 +1692,12 @@ end
         @test !isnothing(Base.Filesystem.getgroupname(s.gid))
     end
 end
+
+@testset "diskstat() works" begin
+    dstat = diskstat()
+
+    # Sanity check assuming disk is smaller than 32TB
+    TB = 2^41
+    @test dstat.total < 32TB
```

Comment:
That's possibly rather small, since current generation disks are already 128TB+, and people might be installing RAID/JBOD for multi-PB disk fs capacities. Let's make it bigger?
```suggestion
    # Sanity check assuming disk is smaller than 32PB
    PB = 2^44
    @test dstat.total < 32PB
```

(Or we could use the ext4+ limitation of 1EB?)

Reply:
done, thanks!